src/olympia/files/tests/test_views.py

```diff
@@ -47,29 +54,38 @@ def setUp(self):
         self.file = self.version.all_files[0]
 
         p = [amo.PLATFORM_LINUX.id, amo.PLATFORM_WIN.id, amo.PLATFORM_MAC.id]
-
         self.file.update(platform=p[0])
 
-        self.files = [self.file,
-                      File.objects.create(version=self.version,
-                                          platform=p[1],
-                                          hash='abc123',
-                                          filename='dictionary-test.xpi'),
-                      File.objects.create(version=self.version,
-                                          platform=p[2],
-                                          hash='abc123',
-                                          filename='dictionary-test.xpi')]
+        files = [
+            (
+                'dictionary-test.xpi',
+                self.file),
+            (
+                'dictionary-test.xpi',
+                File.objects.create(
+                    version=self.version,
+                    platform=p[1],
+                    hash='abc123',
+                    filename='dictionary-test.xpi')),
+            (
+                'dictionary-test-changed.xpi',
+                File.objects.create(
+                    version=self.version,
+                    platform=p[2],
+                    hash='abc123',
+                    filename='dictionary-test.xpi'))]
```

Comment:
This is ugly as hell and I'm open to suggestions but I'd rather not re-work the whole test-case for this bugfix.